editor.planx.uk/src/pages/Preview/Node.tsx

```diff
@@ -155,6 +173,13 @@ const Node: React.FC<Props> = (props) => {
         />
       );
 
+    case TYPES.Filter: {
+      const filterProps = getComponentProps<Filter>();
+      const autoAnswer = nodeId ? autoAnswerableFlag(nodeId) : undefined;
```

Comment:
nit: We could calculate this at the top of the file and add it to both `getComponentProps()` and `PublicProps<T>`. 

Slightly cleaner / more consistent, and we could put a check for component type within `autoAnswerableFlag()` if it's an expensive / slow function.